Makefile

```diff
@@ -27,24 +27,25 @@ include Makefile.common
 .PHONY: defaultentry
 defaultentry: $(DEFAULT_BUILD_TARGET)
 
-include stdlib/StdlibModules
+include $(SRCDIR)/stdlib/StdlibModules
 
 CAMLC = $(BOOT_OCAMLC) $(BOOT_STDLIBFLAGS) -use-prims runtime/primitives
 CAMLOPT=$(OCAMLRUN) ./ocamlopt$(EXE) $(STDLIBFLAGS) -I otherlibs/dynlink
 ARCHES=amd64 arm64 power s390x riscv
-VPATH = utils parsing typing bytecomp file_formats lambda middle_end \
+OC_OCAMLDEPDIRS = utils parsing typing bytecomp file_formats lambda middle_end \
   middle_end/closure middle_end/flambda middle_end/flambda/base_types \
   asmcomp driver toplevel tools
-INCLUDES = $(addprefix -I ,$(VPATH))
+vpath %.cmo $(OC_OCAMLDEPDIRS)
+vpath %.cmi $(OC_OCAMLDEPDIRS)
+vpath %.cmx $(OC_OCAMLDEPDIRS)
```

Comment:
You could have a comment that says what `vpath` is for here and a link to the relevant documentation.